sensor/kubernetes/eventpipeline/pipeline_impl.go

```diff
@@ -90,3 +111,92 @@ func (p *eventPipeline) forwardMessages() {
 		}
 	}
 }
+
+func (p *eventPipeline) processPolicySync(sync *central.PolicySync) error {
+	log.Debug("PolicySync message received from central")
+	return p.detector.ProcessPolicySync(sync)
+}
+
+func (p *eventPipeline) processReassessPolicies() error {
+	log.Debug("ReassessPolicies message received from central")
+	if err := p.detector.ProcessReassessPolicies(); err != nil {
+		return err
+	}
+	if env.ResyncDisabled.BooleanSetting() {
+		message := component.NewEvent()
+		message.AddDeploymentReference(resolver.ResolveAllDeployments(),
+			component.DeploymentRefWithForceDetection(true),
```

Comment:
This will read much better as:
```go
message.AddDeploymentReference(resolver.ResolveAllDeployments(),
			component.WithForceDetection(),
			component.WithSkipResolving())
```

Reply:
Agree.